drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -522,8 +520,8 @@ static int start(struct device *dev, struct gps_config *cfg)
 	}
 
 	if (atomic_get(&drv_data->is_active)) {
-		LOG_WRN("GPS is already active");
-		return -EALREADY;
+		LOG_DBG("GPS is already active. Clean up before restart");
+		cancel_works(drv_data);
```

Comment:
It would be nice to mention it in the API description that `start_gps` will restart the GPS if it is already started. Especially that this changes the behavior of an existing API (even though the previous behavior was not strictly specified).

Reply:
Added comment in gps.h